ros_buildfarm/templates/release/trigger_jobs.groovy.em

```diff
@@ -21,7 +21,7 @@ add_job_names_@(int(i / group_size) + 1)(job_names)
 
 @(FILE('snippet/check_recursive_upstream_projects.groovy'))@
 
-println "Triggering " + job_names.size + " jobs..."
+println "Triggering " + job_names.size() + " jobs..."
```

Comment:
Why does Groovy code need parenthesis for functions without arguments now? If that is the case the same pattern is present in multiple Groovy snippets.

Reply:
> Why does Groovy code need parenthesis for functions without arguments now?

The Groovy sandbox used by the script-security plugin has a whitelist of methods and fields. When running scripts containing that line the sandbox was treating `size` as an invalid field access rather than a method and raising an error. I haven't looked upstream yet to determine if this is a bug or feature.

Reply:
The parentheses here work around a bug in the 1.44 script security plugin. It's referenced in https://issues.jenkins-ci.org/browse/JENKINS-50470 but I think it merits a separate issue as it is still outstanding. I'll open that issue but in the meantime I think it's worth adding the parentheses here. With a comment noting the upstream bug if desired.

It doesn't appear to be generally required to put parentheses on functions, just this specific case.

Reply:
Issue is open upstream https://issues.jenkins-ci.org/browse/JENKINS-54952 @dirk-thomas would you like a comment in source linking to this issue until it's resolved?

Reply:
Not necessary. I was mostly curious why it only happens in this case.